test/api/v3/integration/user/auth/POST-register_local.test.js

```diff
@@ -52,6 +52,22 @@ describe('POST /user/auth/local/register', () => {
       expect(user.tasksOrder.habits).to.have.a.lengthOf(0);
     });
 
+    it('enrolls new users in an A/B test', async () => {
+      let username = generateRandomUserName();
+      let email = `${username}@example.com`;
+      let password = 'password';
+
+      let user = await api.post('/user/auth/local/register', {
+        username,
+        email,
+        password,
+        confirmPassword: password,
+      });
+
+      expect(user._ABtest).to.exist;
```

Comment:
I would still make this attribute private and write a helper function to query the db directly to assert that it exists. There's no need for the client to have this.